src/ontology/cl-edit.owl

```diff
@@ -27002,6 +27010,7 @@ SubClassOf(obo:CL_1000850 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000233
 
 # Class: obo:CL_1000854 (kidney blood vessel cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34843404") obo:IAO_0000115 obo:CL_1000854 "A blood vessel cell that is part of a kidney.")
```

Comment:
That's correct. The logical definition is strange though (very narrow)... Out of the scope of this ticket anyway.

Reply:
yes! new ticket coming right up!